mlrun/model_monitoring/db/tsdb/tdengine/schemas.py

```diff
@@ -148,10 +173,21 @@ def _get_records_query(
                     query.write(f"{timestamp_column} >= '{start}'" + " and ")
                 if end:
                     query.write(f"{timestamp_column} <= '{end}'")
+            full_query = query.getvalue()
+            if full_query.endswith(" and "):
+                full_query = full_query[:-5]
+        if any([limit, interval]):
+            with StringIO() as query:
+                query.write(full_query)
```

Comment:
It doesn't make sense - just write to the same `StringIO` object and call `getvalue` upon completion

Reply:
yeah as Gal mentioned it's an overkill to use StringIO here, so anyway it has been fixed.